core/src/style/polylineStyle.cpp

```diff
@@ -200,6 +200,7 @@ auto PolylineStyleBuilder::parseRule(const DrawRule& _rule, const Properties& _p
         p.stroke.join = static_cast<JoinTypes>(join);
 
         if (!evalWidth(strokeWidth, stroke.width, stroke.slope)) {
+            p.fill.set(fill.width, fill.slope, height, fill.order);
```

Comment:
This is a good catch, the logic for line layout in unfortunately complex :\ Instead of another `set()` could we just move the current `set()` on line 227 to line 187 or so?